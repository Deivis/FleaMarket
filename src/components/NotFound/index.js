import React from 'react';

const NotFound = () => (
  <div className="overlay__not-found">
    <h1> Page not found </h1>
  </div>
);

export default NotFound;
