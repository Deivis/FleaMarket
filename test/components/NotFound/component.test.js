import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../../src/components/NotFound';

describe('Not found Tests', () => {
  test('Not found Render', () => {
    const tree = renderer
      .create(<NotFound />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
