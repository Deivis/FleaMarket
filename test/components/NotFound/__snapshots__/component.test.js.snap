// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found Tests Not found Render 1`] = `
<div
  className="overlay__not-found"
>
  <h1>
     Page not found 
  </h1>
</div>
`;
